### ISSUE TITLE
NPC's can't shoot bow and arrows

### ISSUE THREAD

**Issue by Lentheman7** [TheGameCreators/GameGuruRepo#674]
I can't seem to get any NPC's to shoot with the fantasy bow. I have tried to use NPC models from GG's Character Creator, Fuse/Mixamo/FBX2GG, FPSC, NPC's processed with the model importer with uber soldier anims yet none can shoot arrows. The NPC's will either (1) stand there doing nothing, (2) they stand there acting like they are trying to repeatedly reload, (3) they will walk to the player and kick them, or (4) they slide along the ground with no walking animations. 

If you change the fpe file from hasweapon = fantasy/bow to has weapon = medieval\Bow there is another situation. (1) the bow is invisible, (2) when the NPC fires, a yellow light shines in their face for a second, (3) when the NPC fires it sound like a buzzing noise with a swoosh of the arrow coming at the player, and (4) the arrows seem to start shooting very fast after about 3 to 5 seconds, which quickly kills the player.

I'm not sure if this is related to the gunspec file, the module_combatcore, something that's hard-coded, or some other issue.

**Comment by Pirate-Myke** [comment]
Yes very possible this action will not work on all characters as there are different styles for different purposes. The animations may simply just not be in the model for that weapon style.
